heron/tools/tracker/src/python/tracker.py

```diff
@@ -660,6 +665,10 @@ def setTopologyInfo(self, topology):
     topologyInfo["logical_plan"] = self.extract_logical_plan(topology)
     topologyInfo["physical_plan"] = self.extract_physical_plan(topology)
     topologyInfo["packing_plan"] = self.extract_packing_plan(topology)
+    # extract_packing_plan() was the original implementation. The result is dumped
```

Comment:
Yeah. That is true. I am keeping the old one to be safe just in case it is used anywhere or by anyone. If it is safe enough then I am happy to just replace the old implementation. It is not consistent with the others.